compiler/resolution/functionResolution.cpp

```diff
@@ -1732,16 +1729,16 @@ getInstantiationType(Type* actualType, Type* formalType) {
 static void
 computeGenericSubs(SymbolMap &subs,
                    FnSymbol* fn,
-                   Vec<Symbol*>& formalIdxToActual,
+                   std::map<int, Symbol*>& formalIdxToActual,
                    bool inInitRes) {
   int i = 0;
   for_formals(formal, fn) {
     if (formal->intent == INTENT_PARAM) {
-      if (formalIdxToActual.v[i] && formalIdxToActual.v[i]->isParameter()) {
+      if (formalIdxToActual[i] != NULL && formalIdxToActual[i]->isParameter()) {
```

Comment:
If I understand correctly formalIdxToActual[i] != NULL is a synonym for "does this map have an
element for key "i".  Per our earlier conversation, we believe this will insert a NULL if the map did
not have an entry for "i" which seems unfortunate to me.  I may be missing something but it looks
to me that formalIdxToActual could be declared as const std::map<int, Symbol*>& formalIdxToActual
if it weren't for this.